rust/pact_verifier_cli/src/main.rs

```diff
@@ -268,9 +268,30 @@ fn pact_source(matches: &ArgMatches) -> Vec<PactSource> {
           let auth = matches.value_of("user").map(|user| {
             HttpAuth::User(user.to_string(), matches.value_of("password").map(|p| p.to_string()))
           });
-          PactSource::BrokerUrl(name, s!(v), auth, vec![])
+          let pending = matches.is_present("enable-pending");
+          let wip = match matches.value_of("include-wip-pacts-since") {
```

Comment:
You can just use the map function to change the inner value instead of a pattern match.